purchases/src/main/kotlin/com/revenuecat/purchases/UpgradeInfo.kt

```diff
@@ -5,9 +5,11 @@ import com.android.billingclient.api.BillingFlowParams
 /**
  * This object holds the information used when upgrading from another sku.
  * @property oldProductId The old product ID to upgrade from.
- * @property prorationMode The [BillingFlowParams.ProrationMode] to use when upgrading the given oldSku.
+ * @property prorationMode The [BillingFlowParams.ProrationMode] to use when upgrading the given oldSku. Defaults to
+ * [BillingFlowParams.ProrationMode.IMMEDIATE_WITHOUT_PRORATION].
  */
 data class UpgradeInfo @JvmOverloads constructor(
     val oldProductId: String,
-    @BillingFlowParams.ProrationMode val prorationMode: Int? = null
+    @BillingFlowParams.ProrationMode val prorationMode: Int? =
+        BillingFlowParams.ProrationMode.IMMEDIATE_WITHOUT_PRORATION
```

Comment:
Do we want to change `Int?` to `Int` here so prevent a the Google Console default being used? 🤔 

```suggestion
    @BillingFlowParams.ProrationMode val prorationMode: Int =
        BillingFlowParams.ProrationMode.IMMEDIATE_WITHOUT_PRORATION
```

Reply:
yesss absolutely, good call